lib/rules/lint-attribute-indentation.js

```diff
@@ -379,14 +438,26 @@ module.exports = class AttributeSpacing extends Rule {
 
     return {
       BlockStatement(node) {
-        if (canApplyRule(node, 'BlockStatement', this.config)) {
+        if (canApplyRule(node, node.type, this.config)) {
           this.validateBlockForm(node);
         }
       },
-
       MustacheStatement(node) {
-        if (canApplyRule(node, 'MustacheStatement', this.config)) {
-          this.validateNonBlockForm(node);
+        if (canApplyRule(node, node.type, this.config)) {
+          return this.validateNonBlockForm(node);
+        }
+        return node.loc.end.line;
+      },
+      ElementNode(node) {
+        if (this.config.parseHtmlElements) {
```

Comment:
Can you tweak this to match the flag name? `this.config.processElements`

Reply:
Done as well, now they all match :-).